sql/core/src/test/scala/org/apache/spark/sql/TPCDSSchema.scala

```diff
@@ -0,0 +1,259 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+trait TPCDSSchema {
+
+  private val tableColumns = Map(
+    "call_center" ->
+      """
+        |`cc_call_center_sk` INT, `cc_call_center_id` STRING, `cc_rec_start_date` DATE,
+        |`cc_rec_end_date` DATE, `cc_closed_date_sk` INT, `cc_open_date_sk` INT, `cc_name` STRING,
+        |`cc_class` STRING, `cc_employees` INT, `cc_sq_ft` INT, `cc_hours` STRING,
+        |`cc_manager` STRING, `cc_mkt_id` INT, `cc_mkt_class` STRING, `cc_mkt_desc` STRING,
+        |`cc_market_manager` STRING, `cc_division` INT, `cc_division_name` STRING,
+        |`cc_company` INT, `cc_company_name` STRING, `cc_street_number` STRING,
+        |`cc_street_name` STRING, `cc_street_type` STRING, `cc_suite_number` STRING,
+        |`cc_city` STRING, `cc_county` STRING, `cc_state` STRING, `cc_zip` STRING,
+        |`cc_country` STRING, `cc_gmt_offset` DECIMAL(5,2), `cc_tax_percentage` DECIMAL(5,2)
+        |""".stripMargin,
```

Comment:
Please match the starting and ending indentation of `"""`. You can see it from [the deleted code](https://github.com/apache/spark/pull/25535/files#diff-38fa80d1dc9860f07e135dd02d259269L39).